hphp/runtime/vm/jit/relocation-arm.cpp

```diff
@@ -610,6 +612,7 @@ bool relocatePCRelative(Env& env, TCA srcAddr, TCA destAddr,
         isRelative = false;
       }
     } else if (src->IsUncondBranchImm()) {
+      // Handle both B and BL forms of UncondBranchImm.
       imm >>= kInstructionSizeLog2;
       if (!is_int26(imm) || env.far.count(src)) {
         env.destBlock.setFrontier(destAddr);
```

Comment:
I couldn't exercise this code as is during unit testing or OSS/Mediawiki. However, I forced it by using `if (!is_int6(imm) || env.far.count(src)) {` on the previous line.